third_party/eigen3/gpu_packet_math.patch

```diff
@@ -39,4 +39,127 @@
 +
      return res;
    }
- };
\ No newline at end of file
+ };
+--- a/unsupported/Eigen/SpecialFunctions
++++ b/unsupported/Eigen/SpecialFunctions
+@@ -48,6 +48,9 @@
+ }
+
+ #include "src/SpecialFunctions/SpecialFunctionsImpl.h"
++#if defined(EIGEN_HIPCC)
++#include "src/SpecialFunctions/hip_vector_compatibility.h"
```

Comment:
I suspect we should chance this to UpperCamelCase, to adhere to the coding standards i.e. it should be HipVectorCompatibility.h